src/config/routes/2.1.json

```diff
@@ -201,9 +201,9 @@
         "verbs": ["DELETE"]
     },
     {
-        "path": "/talks/[\\d+]]/starred?",
+        "path": "/talks/[\\d]+/starred/?$",
         "controller": "TalksController",
-        "action": "deleteAction",
+        "action": "deleteTalkStarred",
```

Comment:
Could we rename that function to `unstarTalk`? This `deleteTalkStarred` sounds more like "delete all talks that are starred" (which is – hopefully – not what we do)

Reply:
Storm gave me a squiggly underline on "unstar" so I matched "removeXfromY" seen in at least one other route. lmk what you think